protocol-specs/generators/multisignature/index.js

```diff
@@ -0,0 +1,433 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	hexToBuffer,
+	intToBuffer,
+	hash,
+	getFirstEightBytesReversed,
+	bufferToIntAsString,
+	signData,
+} = require('@liskhq/lisk-cryptography');
+const BaseGenerator = require('../base_generator');
+
+const networkIdentifier =
```

Comment:
can we define this in config or constants of protocol spec folder and it will be useful to reuse other places as well?

Reply:
I think this might change from spec to spec in the future. Maybe is best to keep isolated on each suite? ping @shuse2 ?

Reply:
yes it could differ between spec but also, the test should use the consistent one (doesn't even need to be real), so it should be fine

Reply:
Though we need it to match development as we're using them for test right?

Reply:
depends on where we use i think. If we use it in the unit test for transactions, it shouldn't matter

Reply:
Ok maybe lets create an issue to fix this for all specs?